data/ut/legislature/Jeffrey-D-Stenquist-7201a1bf-a3d2-4d64-9110-103ee78b3989.yml

```diff
@@ -0,0 +1,39 @@
+id: ocd-person/7201a1bf-a3d2-4d64-9110-103ee78b3989
+name: Jeffrey D. Stenquist
+email: jstenquist@le.utah.gov
+image: https://le.utah.gov/images/legislator/STENQJ.jpg
+party:
+- name: Republican
+roles:
+- start_date: 2021-01-01
+  type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
+  district: '46'
+offices:
```

Comment:
@sroomf this person should have a file somewhere, we just need to add their new role:
- start_date: 2023-01-01
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
  district: '46'

They served in house district 51 from jan 2019-jan2023